### PR TITLE
Update o365-data-locations.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated OST To DPA as the DPA is an addendum to the Product Terms (and formerly OST). 

Reference:- https://www.microsoft.com/en-us/licensing/product-licensing/products

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6443

File: microsoft-365/enterprise/o365-data-locations.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Customers should view tenant specific data location information in your Microsof
 
 New Microsoft 365 tenants are defaulted to Geo based on the country of the transaction associated with that tenant's first subscription.
 
-Find information about the contractual commitments for the storage location of customer data at rest in the [Microsoft Online Services Terms (OST)](https://go.microsoft.com/fwlink/p/?LinkId=2098215).
+Find information about the contractual commitments for the storage location of customer data at rest in the [Online Services Data Protection Addendum (DPA)](https://www.microsoft.com/licensing/docs/view/Microsoft-Products-and-Services-Data-Protection-Addendum-DPA).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
Find information about the contractual commitments for the storage location of customer data at rest in the [Microsoft Products and Services Data Protection Addendum (DPA)](https://www.microsoft.com/licensing/docs/view/Microsoft-Products-and-Services-Data-Protection-Addendum-DPA).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dmwmsft Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@dmwmsft Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Kindly help us here. Thanks.

**Comment by tiburd** [comment]
confirmed updated info

**Pull request merged**